blark/iec.lark

```diff
@@ -749,7 +749,7 @@ reset_statement: _variable "R="i expression ";"+
 reference_assignment_statement: _variable "REF="i expression ";"+
 
 // method ::= expression [DEREFERENCED] '.' _identifier '(' ')';
-method_statement: symbolic_variable "(" ")" ";"+
+method_statement: symbolic_variable "(" ")" DEREFERENCED? ";"+
```

Comment:
Looks like it's time to get rid of `method_statement` entirely.

`function_call_statement` (using `function_call`) handles everything it does and more, so having both just makes for ambiguity in the grammar.

I'll make an issue to remove it along with its related `MethodStatement` class.